components/apps/ddm/scenes/Replications.tsx

```diff
@@ -167,26 +168,40 @@ export default function Replications(props: {
           })}
         </div>
       }
+
       {props.replications !== undefined && <PageIndex
         offset={props.getReplicationsConfig.offset || 0}
         onChangeOffset={(offset) => {
           setSearchOffset(offset);
         }}
+        onSearchLimit={(limit) => {
+          setSearchLimit(limit);
+        }}
         limit={props.getReplicationsConfig.limit || 100}
         total={props.replications?.totalCount || 0}
-      />}
+        />}
       {props.replications === undefined && <LoadingIndicator padded />}
     </div>
   );
 }
 
-function PageIndex(props: { offset: number, onChangeOffset: (number) => void, limit: number, total: number }) {
+function PageIndex(props: { offset: number, onChangeOffset: (number) => void, onSearchLimit: (number) => void, limit: number, total: number }) {
   const pageCount = Math.ceil(props.total / props.limit);
-  const currPage = Math.floor(props.offset / props.limit);
+  let currPage = Math.floor(props.offset / props.limit);
 
-  const itemCount = Math.min(props.limit, props.total - props.offset);
-  const firstItem = props.offset + 1;
-  const lastItem = props.offset + itemCount;
+  const limitOptions = [
```

Comment:
in Boost, the user's selection actually get saved in LocalStorage, so they only have to set it once and then the limit will persist until its changed again. Could we do that in DDM UI as well? 

<img width="1156" alt="Screenshot 2023-05-31 at 4 06 08 PM" src="https://github.com/application-research/delta-ui/assets/12418265/b50044e5-58eb-4c77-bbe1-6f702d05a017">

Reply:
I've added the localStorage in latest. Good catch 👍